test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNet/Microsoft.Test.E2E.AspNet.OData.csproj

```diff
@@ -2,66 +2,20 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\..\..\..\sln\packages\xunit.runner.visualstudio.2.3.1\build\net20\xunit.runner.visualstudio.props" Condition="Exists('..\..\..\..\sln\packages\xunit.runner.visualstudio.2.3.1\build\net20\xunit.runner.visualstudio.props')" />
   <Import Project="..\..\..\..\sln\packages\xunit.core.2.3.1\build\xunit.core.props" Condition="Exists('..\..\..\..\sln\packages\xunit.core.2.3.1\build\xunit.core.props')" />
+  <Import Project="..\..\..\..\tools\WebStack.settings.targets" />
   <PropertyGroup>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
     <ProjectGuid>{436F2E52-4CBC-41E5-BD47-66C16184A214}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.Test.E2E.AspNet.OData</RootNamespace>
     <AssemblyName>Microsoft.Test.E2E.AspNet.OData</AssemblyName>
     <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <SccProjectName>SAK</SccProjectName>
-    <SccLocalPath>SAK</SccLocalPath>
-    <SccAuxPath>SAK</SccAuxPath>
-    <SccProvider>SAK</SccProvider>
-    <OutputPath>..\..\..\bin\$(Configuration)\E2ETest\</OutputPath>
-    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\src\</SolutionDir>
-    <RestorePackages>true</RestorePackages>
-    <TargetFrameworkProfile />
+    <OutputPath>$(WebStackRootPath)bin\$(Configuration)\E2ETest\AspNet\</OutputPath>
```

Comment:
This causes the VSTS build to no longer find the test binary. I modified the VSTS task to work for the old and new path by adding \**\.

Reply:
These tests are NOT running in VSTS.

Reply:
Still not running. Looks like the larger change of adding WebStack.settings.targets and removing other stuff, including AssemblyInfo.cs, may have affected this:

```
2018-05-04T16:41:15.3531177Z Starting test execution, please wait...
2018-05-04T16:41:16.1968761Z Warning: [xUnit.net 00:00:01.5174973] Skipping: Microsoft.Test.E2E.AspNet.OData (could not find dependent assembly 'Microsoft.Test.E2E.AspNet.OData, Version=7.0.0')
2018-05-04T16:41:16.1968761Z 
2018-05-04T16:41:16.2593743Z Warning: No test is available in C:\BuildAgent\_work\77\s\bin\Release\E2ETest\AspNet\Microsoft.Test.E2E.AspNet.OData.dll. Make sure that installed test discoverers & executors, platform & framework version settings are appropriate and try again.
```